docs/Using Any API/large-responses.md

```diff
@@ -0,0 +1,120 @@
+
+>❗️ Remember to fund your contract with LINK!
+>
+> Making a GET request will fail unless your deployed contract has enough LINK to pay for it. **Learn how to [Acquire testnet LINK](../acquire-link/) and [Fund your contract](../fund-your-contract/)**.
+> 
+
+<div class="remix-callout">
+    <a href="https://remix.ethereum.org/#optimize=false&evmVersion=null&gist=10ae2499f48f2932b38f34a96b124443&runs=200&version=soljson-v0.8.6+commit.11564f7e.js" target="_blank" class="cl-button--ghost solidity-tracked">Deploy this contract using Remix ↗</a>
+    <a href="../deploy-your-first-contract/" title="">What is Remix?</a>
+</div>
+
+```javascript Kovan
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@chainlink/contracts/src/v0.8/ChainlinkClient.sol";
+
+/**
+ * @notice DO NOT USE THIS CODE IN PRODUCTION. This is an example contract. 
+ */
+contract GenericBigWord is ChainlinkClient {
+  using Chainlink for Chainlink.Request;
+
+  // variable bytes returned in a signle oracle response
+  bytes public data;
+  string public image_url;
+
+  /**
+   * @notice Initialize the link token and target oracle
+   * @dev The oracle address must be an Operator contract for multiword response
+   *
+   *
+   * Kovan Testnet details: 
+   * Link Token: 0xa36085F69e2889c224210F603D836748e7dC0088
+   * Oracle: 0xc57B33452b4F7BB189bB5AfaE9cc4aBa1f7a4FD8 (Chainlink DevRel)
+   *
+   */
+  constructor(
+  ) {
+    setChainlinkToken(0xa36085F69e2889c224210F603D836748e7dC0088);
+    setChainlinkOracle(0xc57B33452b4F7BB189bB5AfaE9cc4aBa1f7a4FD8);
+  }
+
+  ////---- VARIABLE BYTES ARRAY RETURNED BY ORACLE ----////
+
+  /**
+   * @notice Request variable bytes from the oracle
+   */
+  function requestBytes(
+  )
+    public
+  {
+    bytes32 specId = "7a97ff8493ec406d90621b2531f9251a";
+    uint256 payment = 100000000000000000;
+    Chainlink.Request memory req = buildChainlinkRequest(specId, address(this), this.fulfillBytes.selector);
+    req.add("get","https://ipfs.io/ipfs/QmZgsvrA1o1C8BGCrx6mHTqR1Ui1XqbCrtbMVrRLHtuPVD?filename=big-api-response.json");
+    req.add("path", "image");
+    requestOracleData(req, payment);
+  }
+
+  event RequestFulfilled(
+    bytes32 indexed requestId,
+    bytes indexed data
+  );
+
+  /**
+   * @notice Fulfillment function for variable bytes
+   * @dev This is called by the oracle. recordChainlinkFulfillment must be used.
+   */
+  function fulfillBytes(
+    bytes32 requestId,
+    bytes memory bytesData
+  )
+    public
+    recordChainlinkFulfillment(requestId)
+  {
+    emit RequestFulfilled(requestId, bytesData);
+    data = bytesData;
+    image_url = string(data);
+  }
+
+  ////---- END ----////
+}
+```
+
+> The job spec for the Chainlink node in this example can be [found here](../example-job-spec-large/). 
```

Comment:
Please use the following format for callouts:
> 📘 Job Spec
> 
> The job spec for the Chainlink node in this example can be [found here](../example-job-spec-large/).

We're currently using [Readme styling](https://rdmd.readme.io/docs/callouts) so one-line callouts are not highlighted in the docs. Will fix that later.

Reply:
Ahh ok. thanks!